packages/gatsby/cache-dir/static-entry.js

```diff
@@ -221,7 +222,7 @@ export default (pagePath, callback) => {
           as="script"
           rel={script.rel}
           key={script.name}
-          href={urlJoin(pathPrefix, script.name)}
+          href={urlJoin(`scripts`, pathPrefix, script.name)}
```

Comment:
@m-allanson, to confirm, when running `gatsby serve` locally on a build that has `pathPrefix` enabled it shouldn't work? All the assets return 404 because Gatsby is serving the the `public` folder without a prefix (`/blog`)?

Reply:
@brotzky yep, that sounds right. Check out the notes here on how to make this work for testing: https://github.com/gatsbyjs/gatsby/tree/master/examples/using-path-prefix